src/components/ArticleResults.vue

```diff
@@ -8,7 +8,7 @@
                 <li v-for="document in results.records" :key="document.id" class="document">
                     <h3 :data-id="document.id"><a :href="document.url">{{ document.title }}</a></h3>
                     <ul class="metadata">
-                        <li><FormatWithIcon :format="document.type" icon="text"></FormatWithIcon></li>
+                        <li v-if="document.type"><FormatWithIcon :format="document.type" icon="text"></FormatWithIcon></li>
```

Comment:
The `FormatWithIcon` component now requires the `format` prop, so make sure that we have a format in the document before attempting to render it.